### PR TITLE
Stub fixes

### PR DESCRIPTION
**Pull request by xkazuka**
Change or get rid of stubs to be compatible with sparsezoo files v2 stubs

### CODE REVIEW

File: tests/sparseml/pytorch/models/classification/test_mobilenet.py [hunk 1]
```diff
@@ -36,7 +36,6 @@
         ("mobilenet", False, True),
         ("mobilenet", True, False),
         ("mobilenet", "base", False),
-        ("mobilenet", "pruned-conservative", False),
```
Comment (horheynm) [inline tests/sparseml/pytorch/models/classification/test_mobilenet.py:39]: deprecated model

File: tests/sparseml/pytorch/models/classification/test_vgg.py [hunk 2]
```diff
@@ -54,7 +54,6 @@
         ("vgg16", False, True, vgg16),
         ("vgg16", True, False, vgg16),
         ("vgg16", "base", False, vgg16),
-        ("vgg16", "pruned-conservative", False, vgg16),
```
Comment (horheynm) [inline tests/sparseml/pytorch/models/classification/test_vgg.py:57]: deprecated

File: tests/sparseml/pytorch/models/classification/test_resnet.py [hunk 3]
```diff
@@ -63,24 +63,22 @@
         ("resnet50", False, True, resnet50),
         ("resnet50", True, False, resnet50),
         ("resnet50", "base", False, resnet50),
-        ("resnet50", "pruned-conservative", False, resnet50),
-        ("resnet50", "pruned-moderate", False, resnet50),
```
Comment (horheynm) [inline tests/sparseml/pytorch/models/classification/test_resnet.py:67]: both deprecated. stub cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned-moderate, imagenet is not there

File: tests/sparseml/pytorch/models/classification/test_resnet.py [hunk 4]
```diff
@@ -63,24 +63,22 @@
...
         ("resnet50_2xwidth", False, True, resnet50_2xwidth),
-        ("resnet50_2xwidth", True, False, resnet50_2xwidth),
```
Comment (horheynm) [inline tests/sparseml/pytorch/models/classification/test_resnet.py:69]: corr or corr = potentially corrupted file. Might need to re-upload. 
torch.load(model.pth) fails from api, ok from downloading straight from the source

Comment (horheynm) [reply tests/sparseml/pytorch/models/classification/test_resnet.py:69]: models re-uploaded from git server, passes tests

**Review by bfineran — APPROVED** [review]

**Review by Satrat — APPROVED** [review]

**Review by tdg5 — APPROVED** [review]

**Pull request merged**